state/MottattData.ts

```diff
@@ -36,11 +36,11 @@ interface MottattData {
 export default MottattData;
 
 export interface MottatArbeidsgiver {
-  name: string;
-  type: string;
-  parentOrganizationNumber?: string | null;
-  organizationForm: string;
-  organizationNumber: string;
-  socialSecurityNumber?: string | null;
-  status: string;
+  Name: string;
+  Type: string;
+  ParentOrganizationNumber?: string | null;
+  OrganizationForm: string;
+  OrganizationNumber: string;
+  SocialSecurityNumber?: string | null;
+  Status: string;
```

Comment:
Jeg var nok hakket for utydelig på hva de nye feltene skulle hete. For at det skal bli riktig må feltnavnene endres slik:
Name -> navn
Type -> type
OrganizationNumber -> orgnr
OrganizationForm -> orgForm
Status -> status
ParentOrganizationNumber -> orgnrHovedenhet

Reply:
Har oppdatert.